### PR TITLE
makes organs a mapped out variable on species, culling out stuff like mutanteyes, mutantheart etc

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
what the title says, every species has an explicitly filled out list of organs
in addition, we remove organ_status because its useless when synthetic_organ flag exists (also gives it to robotic wings)
also, fixes ethereal heart revives not having their downsides
removes get_availability
changes some regenerate_organs behavior (needs testmerge), adminhealing someone will no longer remove all their unique organs
health analyzer isntead of looking through all your mutantheart mutantlungs and stuff to compare it to defualt values (never fucking worked) instead looks to see if you have some external organs or bodyparts that arent robotics and arent in your species base organs/bodyparts

## Why It's Good For The Game
cool

## Changelog
:cl:
fix: ethereal hearts will now heal stuff they should instead of everything
admin: admin heals will no longer remove unique organs
balance: changes how health analyzer looks for mutants. people with non robotic bodyparts/external organs count now
/:cl:

### CODE REVIEW

**Review by Kapu1178 — CHANGES_REQUESTED** [review]
image

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -86,8 +86,44 @@ GLOBAL_LIST_EMPTY(features_by_species)
...
+
+		ORGAN_SLOT_ZOMBIE = NO_ORGAN,
+		ORGAN_SLOT_PARASITE_EGG = NO_ORGAN,
+
+		ORGAN_SLOT_XENO_HIVENODE = NO_ORGAN,
+		ORGAN_SLOT_XENO_ACIDGLAND = NO_ORGAN,
+		ORGAN_SLOT_XENO_NEUROTOXINGLAND = NO_ORGAN,
+		ORGAN_SLOT_XENO_RESINSPINNER = NO_ORGAN,
+		ORGAN_SLOT_XENO_PLASMAVESSEL = NO_ORGAN,
+		ORGAN_SLOT_XENO_EGGSAC = NO_ORGAN,
```
Comment (Time-Green) [inline code/modules/mob/living/carbon/human/species.dm:125]: This is a massive maintainability hazard. Either make NO_ORGAN implicit, or do what @MrMelbert did in #72080 and turn it into a getter proc that returns the default organ loadout that you can overwrite/adjust so you reduce massive amounts of copypaste.

Comment (Time-Green) [reply code/modules/mob/living/carbon/human/species.dm:125]: I'll review the rest later

**Review by Time-Green — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -509,6 +498,18 @@ GLOBAL_LIST_EMPTY(features_by_species)
 
 	properly_gained = TRUE
 
+/datum/species/proc/get_organ_paths(internal = FALSE, external = FALSE)
```
Comment (Time-Green) [inline code/modules/mob/living/carbon/human/species.dm:501]: Autodoc

**Review by Time-Green — COMMENTED** [review]
Documentation. Code looks fine, gonna conflict a lot though and you still need to change how you store the species organ data (see previous review)

**Comment by Fikou** [comment]
lol

**Comment by Time-Green** [comment]
coward

**Comment by ChungusGamer666** [comment]
soon you'll be revived my friend

soon

**Pull request closed without merging**